tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java

```diff
@@ -98,7 +99,7 @@
  * http://java.sun.com/j2se/1.4.2/docs/api/org/xml/sax/DTDHandler.html
  */
 
-public class CLDRFile implements Freezable<CLDRFile>, Iterable<String>, LocaleStringProvider {
+public class CLDRFile implements Freezable<CLDRFile>, LocaleStringProvider {
```

Comment:
Major change: CLDRFile no longer implements Iterable. Instead, a method must be called to get an iterable/iterator